docs/getting-started/reading-and-writing.mdx

```diff
@@ -108,4 +108,28 @@ async function getUser(username) {
 
     return result.data;
 }
+```
+
+## Private Mode
+
+Private mode allows a node operator to publicly expose its RPC to the internet without the risk of leaking sensitive data within its database. Clients that wish to read data from the node must authenticate with their keypair, allowing access control logic for data within the database to be enforced. This is opposed to “public mode” (the default), where authentication is not required for reads, and thus any client can claim they are any user.
```

Comment:
RPC -> RPC service

Reply:
"thus any client can claim they are any user" is potentially misleading or alarming.  With a view action call this is true, not an action execution that involves a transaction.  But even in the context of a view call, this phrasing kinda implies there's a known vulnerability, but it's basically a way of running a view action with `@caller` set to an arbitrary value (in Kuneiform).  When not in private mode, this is as designed (a public method in Ethereum contracts is runnable in the same way).

Maybe @KwilLuke has a different way to phrase this.

Reply:
Both of these comments are still not resolved.

Reply:
![Screenshot 2024-10-11 at 2 59 31 PM](https://github.com/user-attachments/assets/9616fbc8-1b1e-4058-8093-c31ecbd0681f)

@KwilLuke